src/ol/PluggableMap.js

```diff
@@ -1398,23 +1413,3 @@ function createOptionsInternal(options) {
 
 }
 export default PluggableMap;
-
-/**
- * @param  {Array<import("./layer/Base.js").default>} layers Layers.
- * @return {boolean} Layers have sources that are still loading.
- */
-function getLoading(layers) {
-  for (let i = 0, ii = layers.length; i < ii; ++i) {
-    const layer = layers[i];
-    if (typeof /** @type {?} */ (layer).getLayers === 'function') {
```

Comment:
This was supposed to handle layer groups, right? Will they still be handled with the updated code?

Reply:
The `layerStatesArray` is already flattened and contains all layers from all groups of the root group.